### ISSUE TITLE
Warnings with relative paths not parsed

### ISSUE THREAD

**Issue by dkt01** [JacobDomagala/CompileResult#31]
In my project, some warning messages include an absolute path and others have a relative path.  CompileResult only parses the warnings with an absolute path.

Example of a warning that is detected:
```
Checking line: /__w/app_cfh/app_cfh/prod/1A/plat_planning_nav_impl/planning_ros/src/control_conversions.cpp:86:13: warning: narrowing conversion of ‘(__gnu_cxx::__alloc_traits<std::allocator<double>, double>::value_type)message.planning_msgs::ControlImplJoint_<std::allocator<void> >::falling_effort_rate_limits.std::vector<double, std::allocator<double> >::operator[](i)’ from ‘__gnu_cxx::__alloc_traits<std::allocator<double>, double>::value_type’ {aka ‘double’} to ‘float’ [-Wnarrowing] with idx=0 true and true
```

Example of a warning that is not detected:
```
Checking line: prod/common/newmat/newmat5.cpp:494:69: warning: throw will always call terminate() [-Wterminate] with idx=-1 true and true
```

The file `prod/common/newmat/newmat5.cpp` is at absolute path `/__w/app_cfh/app_cfh/prod/common/newmat/newmat5.cpp`.

This appears to be because the absolute path `prefix_dir` is not at the beginning of the log lines ([ref](https://github.com/JacobDomagala/CompileResult/blob/f144262f7755d7ba6b3699b285c8a811c966b173/dist/index.js#L188))
